benchmarks/infrastructure/gcp/terraform/README.md

```diff
@@ -0,0 +1,46 @@
+# Create infrastructure with Terraform
+
+1. Install [Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli?in=terraform/gcp-get-started).
+
+2. Create and download [service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#iam-service-account-keys-create-console).
```

Comment:
I would add: key file should be in JSON format

Reply:
done